fixtures/uking/mini-spider-light.json

```
@@ -0,0 +1,317 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Mini Spider Light",
+  "shortName": "UK-MiniSpider",
```

The fixture name on the product page includes `ZQ-B20`, which is also the manual's file name. So I think it should be included in the name:

```suggestion
  "name": "ZQ-B20 Mini Spider Light",
  "shortName": "ZQ-B20-MiniSpider",
```

Please also rename the file to `zq-b20-mini-spider-light.json`.